slam-ws/src/load_waypoints/scripts/load_waypoints_server.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import rospy, os, json 
+from load_waypoints.srv import *
+
+all_waypoints = list()
+
+def parse_json():
+    base_dir = rospy.get_param('~arg_name') # ~ added to arg_name because private param 
+    
+    with open(base_dir + '/scripts/waypoints.json') as f:
+        data = json.load(f)
+
+    # Parse through dictionary and create list of lists holding all waypoints
+    for waypoint in data["waypoints"]:
+        all_waypoints.append([waypoint['coordinate 1'], waypoint['coordinate 2']])
+
+    print("Loaded waypoints:", all_waypoints)
+    return 
+
+def handle_waypoint_number(req):
+    print ("Returning request for waypoint #%s "%(req.waypoint_number))
+    return LoadWaypointResponse(all_waypoints[req.waypoint_number])
+
+def load_waypoints_server():
+    # Must init node before reading any files
+    rospy.init_node('load_waypoints_server')
```

Comment:
The node itself is a `waypoint_server`, so let's rename it from `load_waypoints_server`